libs/hooks/open-telemetry/src/lib/open-telemetry-hook.ts

```diff
@@ -8,33 +8,33 @@ import { Span, Tracer, trace } from '@opentelemetry/api';
 
 const SpanProperties = Object.freeze({
   FLAG_KEY: 'feature_flag.flag_key',
-  CLIENT_NAME: 'feature_flag.client.name',
-  CLIENT_VERSION: 'feature_flag.client.version',
-  PROVIDER_NAME: 'feature_flag.provider.name',
-  VARIANT: 'feature_flag.evaluated.variant',
-  VALUE: 'feature_flag.evaluated.value',
+  PROVIDER_NAME: 'feature_flag.provider_name',
+  VARIANT: 'feature_flag.evaluated_variant',
+  VALUE: 'feature_flag.evaluated_value',
 });
 
 export class OpenTelemetryHook implements Hook {
   private spanMap = new WeakMap<HookContext, Span>();
   private tracer: Tracer;
 
-  constructor(name: string, version?: string) {
-    this.tracer = trace.getTracer(name, version);
+  constructor() {
+    this.tracer = trace.getTracer(
+      '@openfeature/open-telemetry-hook',
+      '1.0.6-alpha' // x-release-please-version
```

Comment:
Release Please should automatically update this value.

Reply:
This is very cool, if it works like it seems.